Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8.12
+# not sure what these two lines do but they were in
+# an example I found.
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
```

Comment:
all output for python commands will not be buffered prior to being sent to standard out, not too sure if this is degrades performance, but if they were in the example you found that's good enough for me.